seedbuilder/areas.ori

```diff
@@ -1564,52 +1564,56 @@ home: GinsoOuterDoor 527 -42
 	-- anchor: just outside the ginso door, unlocked
 	conn: GinsoInnerDoor
 	conn: OuterSwampUpperArea
-home: GinsoInnerDoor 526 2
+home: GinsoInnerDoor 522 1
 	-- anchor: Just inside the Ginso tree door after putting in the water vein
 	conn: LowerGinsoTree
-		casual WallJump DoubleJump
-		casual Climb DoubleJump
+		casual DoubleJump WallJump
+		casual DoubleJump Climb
 		casual ChargeJump
 		casual Bash Grenade
 		master DoubleJump
-		master ChargeDash WallJump
-		master ChargeDash Climb
+		master ChargeDash WallJump Energy=2
+		master ChargeDash Climb Energy=2
 	conn: GinsoOuterDoor
 home: LowerGinsoTree 510 100
 	-- anchor: At the top of the second room near the flower
 	pickup: LowerGinsoHiddenExp
-		casual ChargeJump
-		casual Bash Grenade
 		casual WallJump DoubleJump
 		casual Climb DoubleJump
+		casual ChargeJump
+		casual Bash Grenade
 		master DoubleJump
-		master ChargeDash Climb
-		master ChargeDash WallJump
+		master ChargeDash Climb Energy=1
+		master ChargeDash WallJump Energy=1
 	pickup: LowerGinsoPlant
 		casual ChargeFlame
 		casual Grenade
-		expert ChargeDash
+		expert ChargeDash Energy=1
+	conn: GinsoInnerDoor
 	conn: GinsoMiniBossDoor
 		casual WallJump DoubleJump
 		casual WallJump Glide
 		casual Climb DoubleJump
 		casual Climb Glide
-		casual ChargeJump Glide
 		casual ChargeJump DoubleJump
+		casual ChargeJump Glide
 		casual Bash Grenade DoubleJump
 		casual Bash Grenade Glide
 		standard WallJump Health=3
 		standard Climb Health=3
```

Comment:
```suggestion
		expert Climb Health=3
```
Requires 2 dboosts in standard:
- One in the lower route
- One in the TP room
In expert you expect the player to avoid the second dboost